doit/cmd_info.py

```diff
@@ -12,7 +12,7 @@
     'short': 's',
     'long': 'status',
     'type': bool,
-    'default': False,
+    'default': True,
```

Comment:
I am OK with changing the default. But boolean values must have default value `False`, otherwise it is not possible to set their value to `False`. In this case I think better add another parameter like `hide-status`.

Reply:
Good point. Maybe `no-status` which seems a more widely used convention ?

Reply:
ok. `no-status`